core/src/test/java/tachyon/master/LocalTachyonClusterMultiMaster.java

```diff
@@ -205,9 +200,8 @@ public void start() throws IOException {
     }
 
     mWorker =
-        TachyonWorker.createWorker(
-            CommonUtils.parseInetSocketAddress(mCuratorServer.getConnectString()),
-            new InetSocketAddress(mLocalhostName, 0), 0, 1, 100, mWorkerConf);
+        TachyonWorker.createWorker(mCuratorServer.getConnectString().split(":")[0],
```

Comment:
yes, moving to TachyonWorker.createWorker(TachyonConf) means we have to redo the work which is done by TachyonWorker.createWorker(String masterAddress, int masterPort, int dataPort, int minWorkerThreads, int maxWorkerThreads, TachyonConf tachyonConf) here.